towerlib/entities/job.py

```diff
@@ -1986,6 +1997,53 @@ def module_name(self):
         """
         return self._data.get('module_name')
 
+
+class WorkflowNodes(Entity):
+    """Models the user entity of ansible tower."""
+
+    def __init__(self, tower_instance, data):
+        Entity.__init__(self, tower_instance, data)
+
+    @property
+    def count(self):
+        """The job this event belongs to.
```

Comment:
Can we please update the docstrings to match the usage? Also on the other properties?

Reply:
Done.